libs/lib-client/core/src/services/feature-toggles.ts

```diff
@@ -0,0 +1,35 @@
+import { Injectable, APP_INITIALIZER, FactoryProvider } from '@angular/core';
+import { tap, catchError } from 'rxjs/operators';
+import { EMPTY } from 'rxjs';
+import { RestApiService } from './restapi';
+
+@Injectable({ providedIn: 'root' })
+export class FeatureToggleService {
+  features: { [featureName: string]: any };
+
+  isFeatureEnabled(featureName: string) {
+    return this.features && !!this.features[featureName];
+  }
+}
+
+export function featureToggleServiceFactory(
```

Comment:
Should we need to export this function if it is only used in this file?

Reply:
turns out it was needed for AoT process